NickvisionTubeConverter.Shared/Controllers/MainWindowController.cs

```diff
@@ -89,9 +90,17 @@ public class MainWindowController : IDisposable
     public MainWindowController(string[] args)
     {
         _disposed = false;
+        _noNetCheck = false;
         if (args.Length > 0)
         {
-            UrlToLaunch = args[0];
+            if (args[0] == "--no-net-check")
+            {
+                _noNetCheck = true;
+            }
```

Comment:
I thinks this would better be the Aura feature controlled by env var: if AURA_DISABLE_NETCHECK (or something like this) is set, then return status as connected and don't watch network changes. Env var is preferred IMO because it's easier to modify for flatpak than launch arguments.

Reply:
Well as I said in the commit message, I don't really see this being used in production cases.

it's just meant for me to do a quick `dotnet run --no-net-check` or `flatpak run org.nickvision.tubeconverter//master --no-net-check` in WSL when building the Linux version.

Reply:
ah, that's true.

let me whip up something in Aura and make a PR

Reply:
> let me whip up something in Aura and make a PR

Here we go: https://github.com/NickvisionApps/Aura/pull/40